Code/Mantid/Framework/DataHandling/src/SaveOpenGenieAscii.cpp

```diff
@@ -170,5 +200,73 @@ void SaveOpenGenieAscii::writeAxisValues(std::string alpha,
   }
 }
 
+//-----------------------------------------------------------------------
+/** Reads the sample logs
+   *  @param outfile :: File it will save it out to
+   *  @param fourspc :: Constant string for four spaces
+   */
+void SaveOpenGenieAscii::writeSampleLogs(std::ofstream &outfile,
+                                         std::string fourspc) {
+  const std::vector<Property *> &logData = ws->run().getLogData();
+
+  // std::vector<std::string> myVector;
+  for (auto log = logData.begin(); log != logData.end(); ++log) {
+    std::string name = (*log)->name();
+    std::string type = (*log)->type();
+    std::string value = (*log)->value();
+
+    if (type.std::string::find("vector") &&
+        type.std::string::find("double") != std::string::npos) {
+
+      auto tsp = ws->run().getTimeSeriesProperty<double>(name);
+      value = std::to_string(tsp->timeAverageValue());
```

Comment:
On rhel6 this is producing (http://builds.mantidproject.org/job/pull_requests-rhel6/2557/console):
```
15:49:51 /home/builder/jenkins-linode/workspace/pull_requests-rhel6/Code/Mantid/Framework/DataHandling/src/SaveOpenGenieAscii.cpp:222: error: call of overloaded ‘to_string(double)’ is ambiguous
15:49:51 /usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/bits/basic_string.h:2604: note: candidates are: std::string std::to_string(long long int)
15:49:51 /usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/bits/basic_string.h:2610: note:                 std::string std::to_string(long long unsigned int)
15:49:51 /usr/lib/gcc/x86_64-redhat-linux/4.4.7/../../../../include/c++/4.4.7/bits/basic_string.h:2616: note:                 std::string std::to_string(long double)
```

So you would need to `static_cast<double>` the time average value, or use boost::lexical_cast<std::string> as an alternative.

Same happens below: lines 229 (int to string), and 236 (bool to string).